src/main/scala/beam/agentsim/agents/ridehail/repositioningmanager/DemandFollowingRepositioningManager.scala

```diff
@@ -81,8 +81,11 @@ class DemandFollowingRepositioningManager(val beamServices: BeamServices, val ri
     createClusters
   }
 
-  def repositionVehicles(tick: Int): Vector[(Id[Vehicle], Location)] = {
-    val nonRepositioningIdleVehicles = rideHailManager.vehicleManager.getIdleVehiclesAndFilterOutExluded.values
```

Comment:
Same as above - we shouldn't push this to the calling method - unless it is NOT an implementation detail.

Reply:
We're repositioning only idle vehicles so it seems ok for me to pass the vehicles to this method.